tests/testthat/test-derive_var_atoxgr.R

```diff
@@ -3191,3 +3191,2836 @@ test_that("derive_var_atoxgr, test 81: CTCAEv4 Hypophosphatemia", {
+
+### Grade 4: >=3 x ULN
+### Grade 3: 2 to <3 x ULN
+### Grade 2: 1.5 to < 2 x ULN
+### Grade 1: 1.1 to < 1.5 x ULN
+
+## Test 118: DAIDS INR High ----
+test_that("derive_var_atoxgr Test 118: DAIDS INR High", {
+  expected_inri_daids <- tibble::tribble(
+    ~ATOXDSCH,     ~AVAL,  ~ANRHI, ~AVALU,         ~ATOXGRH,
+    "Not a term",  80,     80,     NA_character_,  NA,
+    NA_character_, 60,     80,     NA_character_,  NA,
+    "INR, High",   240,    80,     NA_character_,  "4",
+    "INR, High",   239,    80,     NA_character_,  "3",
+    "INR, High",   160,    80,     NA_character_,  "3",
+    "INR, High",   159,    80,     NA_character_,  "2",
+    "INR, High",   120,    80,     NA_character_,  "2",
+    "INR, High",   119,    80,     NA_character_,  "1",
+    "INR, High",   88,     80,     NA_character_,  "1",
+    "INR, High",   87,     80,     NA_character_,  "0",
+    # ANRHI missing - cannot grade
+    "INR, High",   100,    NA,     NA_character_,  NA,
+    # AVAL missing cannot grade
+    "INR, High",   NA,     80,     NA_character_,  NA,
+  )
+
+  input_inri_daids <- expected_inri_daids %>%
+    select(-ATOXGRH)
+
+  actual_inri_daids <- derive_var_atoxgr_dir(
+    input_inri_daids,
+    new_var = ATOXGRH,
+    meta_criteria = atoxgr_criteria_daids,
+    tox_description_var = ATOXDSCH,
+    criteria_direction = "H",
+    get_unit_expr = AVALU
+  )
+
+  expect_dfs_equal(
+    base = expected_inri_daids,
+    compare = actual_inri_daids,
+    keys = c("ATOXDSCH", "AVAL", "ANRHI", "AVALU")
+  )
+})
+
+
+### Methemoglobin
+
+### Grade 4: >=20.0%
+### Grade 3: 15 to < 20%
+### Grade 2: 10 to < 15%
+### Grade 1: 5 to <10%
+
+## Test 119: DAIDS Methemoglobin ----
+test_that("derive_var_atoxgr Test 119: DAIDS Methemoglobin", {
+  expected_methi_daids <- tibble::tribble(
+    ~ATOXDSCH,         ~AVAL, ~AVALU,  ~ATOXGRH,
+    "Not a term",      20,    "%",     NA,
+    NA_character_,     20,    "%",     NA,
+    "Methemoglobin",   20,    "%",     "4",
+    "Methemoglobin",   19,    "%",     "3",
+    "Methemoglobin",   15,    "%",     "3",
+    "Methemoglobin",   14,    "%",     "2",
+    "Methemoglobin",   10,    "%",     "2",
+    "Methemoglobin",   9,     "%",     "1",
+    "Methemoglobin",   5,     "%",     "1",
+    "Methemoglobin",   4,     "%",     "0",
+    # Unit wrong - cannot grade
+    "Methemoglobin",   100,   NA,      NA,
+    # AVAL missing cannot grade
+    "Methemoglobin",   NA,    "%",     NA,
+  )
+
+  input_methi_daids <- expected_methi_daids %>%
+    select(-ATOXGRH)
+
+  actual_methi_daids <- derive_var_atoxgr_dir(
+    input_methi_daids,
+    new_var = ATOXGRH,
+    meta_criteria = atoxgr_criteria_daids,
+    tox_description_var = ATOXDSCH,
+    criteria_direction = "H",
+    get_unit_expr = AVALU
+  )
+
+  expect_dfs_equal(
+    base = expected_methi_daids,
+    compare = actual_methi_daids,
+    keys = c("ATOXDSCH", "AVAL", "AVALU")
+  )
+})
+
+### PTT, high
+
+### Grade 4: >=3 x ULN
+### Grade 3: 2.33 to <3 x ULN
```

Comment:
can't help but feel the intention was for "2 and a third", and "1 and two-thirds", but the official doc says these decimal cutoffs so we can certainly leave it, similar to your interpretations of something like `1.499` in WBC, the case when uses a cutoff of inequalities pertaining to 1.5

Reply:
Yeah I agree, but need to follow doc.